charts/security-apps/Chart.yaml

```diff
@@ -18,9 +18,11 @@ annotations:
   artifacthub.io/changes: |
     - kind: changed
       description: |
-        chore: update Stackrox Helm chart from 400.6.2 to 400.8.0
+        chore: Bump Kyverno from 1.4.4 to 1.5.1
```

Comment:
Does the update contain any changes that are relevant for our use cases?

Reply:
I have updated the descriptions, but it seems the only relevant change for us is the bitnami change, I hope I didnt miss anything.